yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -12,375 +12,181 @@
-<section anchor="file_names" title="File names">
-<t>This section updates <xref target="RFC7950"/>
- section 5.2, <xref target="RFC6020"/>
- section 5.2 and <xref target="RFC8407"/>
- section 3.2</t>
-<t>If a revision has an associated revision label, then it is RECOMMENDED that the name of the file for that revision be of the form:
-<figure align="center" anchor="filename" suppress-title="true">
-<artwork align="left" name="filename-abnf">
-<![CDATA[
-  module-or-submodule-name ['#' revision-label] ( '.yang' / '.yin' )
-
-    E.g., acme-router-module#2.0.3.yang
-]]>
-</artwork>
-</figure>
-      YANG module (or submodule) files may be identified using either the revision-date (as per <xref target="RFC8407"/>
- section 3.2) or the revision label.
-</t>
-</section>
-
-<section anchor="revision_label-scheme" title="Revision label scheme extension statement">
-<t>The optional "rev:revision-label-scheme" extension statement is used to indicate which revision label
-            scheme a module or submodule uses. There MUST NOT be more than one revision label scheme in a module or submodule. The mandatory argument to this extension statement:
-<list style="symbols">
-<t>specifies the revision label scheme used by the module or submodule</t>
-<t>is defined in the document which specifies the revision label scheme</t>
-<t>MUST be an identity derived from "revision-label-scheme-base".</t>
-</list>
-</t>
-<t>The revision label scheme used by a module or submodule SHOULD NOT change during the
-         lifetime of the module or submodule. If the revision label scheme used by a module or submodule
-         is changed to a new scheme, then all revision label statements that do not
-         conform to the new scheme MUST be replaced or removed.</t>
-</section>
-</section>
-
-<section anchor="example_versions" title="Examples for updating the YANG module revision history">
-<t>The following diagram, explanation, and module history illustrates how the branched revision history,
-    "non-backwards-compatible" extension statement, and revision "label" extension statement could be used:</t>
-<figure>
-<preamble>Example YANG module with branched revision history.</preamble>
-<artwork>
-       Module revision date        Revision label
+        </figure>
+        <t>In the revision history example above (with revision descriptions elided for clarity), removing the revision history entry for 2020-02-10 would also remove the rev:non-backwards-compatible annotation and hence the resulting revision history would incorrectly indicate that revision 2020-06-07 is backwards-compatible with revisions 2019-01-02 through 2019-10-21 when it is not, and so this change cannot be made.  Conversely, removing one or more revisions out of 2019-03-04, 2019-10-21 and 2020-08-09 from the revision history would still retain a consistent revision history, and is acceptable, subject to an awareness of the concerns raised in the first paragraph of this section.</t>
```

Comment:
elided?

Reply:
You prefer removed? That change was done by @rgwilton IIRC.

Reply:
Will change to omitted